module.exports = {
    INTERNAL_ERROR: 'An internal error occurred, please try again later'
}